bookwyrm/templates/lists/list.html

```diff
@@ -40,6 +81,15 @@
                             <input type="hidden" name="item" value="{{ item.id }}">
                             <button type="submit" class="button is-small is-danger">{% trans "Remove" %}</button>
                         </form>
+                    </div>
+                    <div class="card-footer has-background-white-bis">
+                        <div>
+                            <form name="set-position" method="post" action="{% url 'list-set-book-position' item.id %}" class="card-footer-item">
+                                {% csrf_token %}
+                                <input class="input" type="text" name="position" value="">
```

Comment:
@mouse-reeve I am not sure how to address this. Advice or suggestion?

Reply:
You can add a `label` element with the class `is-sr-only` and it will only be visible to screen readers. I think you can also use the `aria-label` attribute on the `input` element.

Reply:
Thanks. This is a great thing to have in CI!